app/models/book_authorship.rb

```diff
@@ -1,2 +1,4 @@
 class BookAuthorship < ApplicationRecord
+  belongs_to :books, class_name: Book.name, foreign_key: :book_id
+  belongs_to :authors, class_name: Author.name, foreign_key: :author_id
```

Comment:
has_many mới số nhìu, còn belogn to số ít mà 
```suggestion
  belongs_to :book
  belongs_to :author
```

Reply:
Cảm ơn chị, cái này em chưa biết